### DISCUSSION TITLE
the tool remote control function

### DISCUSSION THREAD

**Discussion by BalteMD** [winsiderss/systeminformer#1522] [Questions / Answers]
Can someone explain to me what is the tool remote control function?
![image](https://user-images.githubusercontent.com/92046606/205452382-10eef713-7b9b-45d6-b767-e725d6a737cb.png)

**Comment by dmex** [comment]
> Can someone explain to me what is the tool remote control function?

The "remote control" feature is called _Shadowing_ in Microsoft documentation. By default the existing desktop session is locked when another user connects remotely but shadowing allows both users to share and interact within the same desktop session.
